### ISSUE TITLE
Random test generation

### ISSUE THREAD

**Issue by OYounis** [riscv/sail-riscv#102]
Hello all, 
According to the Readme, Sail can be used for random test generation. But I cannot find how to do so.

**Comment by jrtc27** [comment]
That uses https://github.com/rems-project/sail-riscv-test-generation

**Comment by OYounis** [comment]
Is there any documentation on how to use it?

**Comment by bacam** [comment]
There isn't much, although I did add a bit to the Sail manual about the underlying feature.  This is a pretty simple form of test generation, though: it just produces random instructions from the model for use with [the TestRIG tool](https://github.com/CTSRD-CHERI/TestRIG/).  We have used Sail to produce more self-contained random tests for another architecture using an accompanying symbolic execution tool, but not currently for RISC-V.
